opentelemetry-proto/src/transform/logs.rs

```diff
@@ -98,14 +98,22 @@ pub mod tonic {
                             }),
                         })
                         .collect();
-                    if let Some(event_name) = log_record.event_name.as_ref() {
-                        attributes.push(KeyValue {
-                            key: "name".into(),
-                            value: Some(AnyValue {
-                                value: Some(Value::StringValue(event_name.to_string())),
-                            }),
-                        });
+                    #[cfg(feature = "populate-logs-event-name")]
+                    {
```

Comment:
Remove the extra set of braces wrapping the `if/else` block? You don't have that for the stdout exporter.

Reply:
yeah it's weird but I got the lint error without the braces - https://github.com/open-telemetry/opentelemetry-rust/actions/runs/10292895062/job/28488187666?pr=1994